lib/src/config/cli/arguments_parser.dart

```diff
@@ -108,7 +113,14 @@ void _appendMetricsThresholdOptions(ArgParser parser) {
       valueHelp: '${metric.defaultValue}',
       // ignore: avoid_types_on_closure_parameters
       callback: (String i) {
```

Comment:
Could `i` be renamed to something more meaningful?

Reply:
fix